tfhe/src/high_level_api/keys/server.rs

```diff
@@ -1,6 +1,10 @@
 #[cfg(feature = "gpu")]
 use crate::core_crypto::gpu::{synchronize_devices, CudaStreams};
 use crate::high_level_api::keys::{IntegerCompressedServerKey, IntegerServerKey};
+use crate::shortint::glwe_compression::{
+    CompressedGlweCompressionKey, CompressedGlweDecompressionKey, CompressionKey,
```

Comment:
Glwe naming

Reply:
problem is it's inconsistent at the moment, as you can see on this line, so let's remove all GLWE naming above core when it's in type and function names.

you can squash your commits if you want, not need to keep the structure if it's slowing you down

Reply:
also we have shortint primitives at the HL level ?

Reply:
> also we have shortint primitives at the HL level ?

I don't think so but it's possible

>  so let's remove all GLWE naming above core when it's in type and function names

Will do

Reply:
Done